client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx

```diff
@@ -17,12 +18,15 @@ import useStyles from './ContactQuestioningStyles';
 import { ADDITIONAL_PHONE_LABEL } from '../PersonalInfoTab/PersonalInfoTab';
 
 const ContactQuestioningPersonal: React.FC<Props> = (props: Props): JSX.Element => {
-    
-    const { interactedContact, changeIdentificationType, updateInteractedContact } = props;
+    const {control , getValues} = useFormContext();
+
+    const { index , interactedContact } = props;
     
     const [shouldIdDisable, setShouldIdDisable] = useState<boolean>(false);
     
-    const { isFieldDisabled } = useContactFields(interactedContact.contactStatus);
+    const formValues = getValues().form ? getValues().form[index] : interactedContact;
+    const { isFieldDisabled } = useContactFields(formValues.contactStatus);
+    const [idType , setIdType] = useState<boolean>(formValues.identificationType === IdentificationTypes.PASSPORT);
```

Comment:
I think the name isnt fit, because boolean can be an id type, and the fact that true is passport isnt so clear

Reply:
fixed [here](
7730b19)